### DISCUSSION TITLE
Contributing to Biometrics Work

### DISCUSSION THREAD

**Discussion by shriyajad** [sensein/senselab#111] [Ideas]
Hi, I'm Shriya Jaddu, a rising senior at Langley High School in Virginia. I recently presented my work at the NIH Brain Initiative Conference, where I came across Dr. Ghosh's research. I am very interested in contributing to this work, and Dr. Ghosh suggested submitting a discussion post here. I have experience working with image classification, audio classification, and time-series classification in medical applications such as breast cancer prediction and seizure forecasting. I have also worked with data augmentation with GANs. Please let me know if there is anything I can help with! I am up to help with anything needed!

- Shriya

**Comment by fabiocat93** [comment]
Hi @shriyajad ,

Thanks so much for reaching out (and thanks, @satra, for pointing you to our repo)! 
We currently have some exciting projects, including some efforts around active speaker detection from audio and video (figuring out who is speaking and when). While our current work is primarily audio-based, we're looking to expand into video and multimodal analysis. We also have some datasets that need curating, which could greatly benefit from this work.

Does this sound interesting to you? We'd love to hear your thoughts!

Fabio

**Reply by shriyajad** [reply]
Hi @fabiocat93, 

Thank you so much for getting back to me! Your work with active speaker detection sounds really cool. Expanding this to video and multimodal analysis also sounds fascinating. I would love to help with curating the datasets, so pease let me know what I can do to help!

Best,
Shriya

**Reply by fabiocat93** [reply]
hi @shriyajad, are you available for e-meeting next week and discuss further the project? You can email me at fabiocat@mit.edu

**Reply by shriyajad** [reply]
Hi @fabiocat93, 

Yes, I would love to meet to discuss the project further! I reached out directly to your email from my email address (shriyajaddu@gmail.com) about when I'm available next week.

Best,
Shriya

**Reply by fabiocat93** [reply]
Hi @shriyajad ,

Thanks for your time! As we discussed in the meeting, the next steps involve exploring tools like [Mediapipe](https://mediapipe-studio.webapps.google.com/home) and [DeepFace](https://github.com/serengil/deepface/tree/master). You can start by analyzing any Zoom recordings from both the speaker and gallery view. Also, feel free to join our biometrics meetings—I just sent you a Zoom invite.

Best,
Fabio

**Reply by shriyajad** [reply]
Hi Dr. Catania, 

Thank you so much for your time chatting with me. I will look into these tools and let you know if I have any questions or find anything. 

Best,
Shriya
